index.html

```diff
@@ -4264,6 +4264,48 @@ <h3>Spectrum of Privacy</h3>
         </p>
       </section>
 
+      <section class="informative">
+        <h3>Software Trust Boundaries</h3>
+
+        <p>
+There are a variety of trust boundaries that exist in the
+<a href="#ecosystem-overview">ecosystem described by this specification</a>.
+Just like an individual using a web browser trusts the web browser not to betray
+their trust by uploading their personal information to a data broker, a similar
+trust exists between the roles in the ecosystem described by this specification
+and the software that operates on the behalf of each of those roles. For
+example,
+        </p>
+
+        <ul>
+          <li>
+An <a>issuer</a> trusts the issuer software it uses to not issue <a>verifiable
```

Comment:
```suggestion
An <a>issuer's</a> <a href="https://www.w3.org/WAI/UA/2011/ED-UAAG20-20110525/#def-user-agent">User agent</a> is expected to not issue <a>verifiable
```

Reply:
Updated to use latest ref: https://www.w3.org/TR/UAAG20/#def-user-agent